models/turbine_models/custom_models/pipeline_base.py

```diff
@@ -238,6 +243,41 @@ def __call__(self, function_name, inputs: list):
         return output
 
 
+class Printer:
```

Comment:
Is there a reason to use this instead of just import logging and use that?

Reply:
We can set it up as a logger, I used this since we had it setup nicely for tresleches full_runner.py